File: engine/model/src/main/scala/tools/packet/playerinteraction/PlayerShopErrorMessage.scala
package tools.packet.playerinteraction

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class PlayerShopErrorMessage(private var _error: Int, private var _type: Int) extends PacketInput {
  def error: Int = _error

  def theType: Int = _type

  override def opcode(): SendOpcode = SendOpcode.PLAYER_INTERACTION
}

File: engine/engine-model/src/main/java/tools/packet/character/interaction/PlayerShopErrorMessage.java
package tools.packet.character.interaction;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record PlayerShopErrorMessage(Integer error, Integer theType) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PLAYER_INTERACTION;
   }
}